server/controllers/getMeals.js

```diff
@@ -0,0 +1,16 @@
+const getMealsByCat = require('../database/queries/getMealsByCat');
+
+const getMeals = (req, res, next) => {
+  let { category } = req.query;
```

Comment:
Add a default value for the category incase the URL params are not supplied
```suggestion
  let { category = 'main' } = req.query;
```